sklearn/ensemble/_hist_gradient_boosting/tests/test_warm_start.py

```diff
@@ -166,15 +168,16 @@ def _get_rng(rng_type):
             return np.random.RandomState(0)
 
     random_state = _get_rng(rng_type)
-    gb_1 = GradientBoosting(n_iter_no_change=5, max_iter=2,
-                            random_state=random_state)
+    gb_1 = GradientBoosting(n_iter_no_change=5, early_stopping=True,
```

Comment:
remove `n_iter_no_change`?

Reply:
Yes! However I have to add `scoring`, since the new default is `'loss'` and the subsample set is only computed if `scoring != 'loss'`